ipatests/test_integration/test_external_ca.py

```diff
@@ -21,12 +21,13 @@
 from ipatests.pytest_plugins.integration import tasks
 from ipatests.test_integration.base import IntegrationTest
 from ipaplatform.paths import paths
-
+import os
 from itertools import chain, repeat
+from ipatests.pytest_plugins.integration.create_external_ca import ExternalCA
 
 IPA_CA = 'ipa_ca.crt'
 ROOT_CA = 'root_ca.crt'
-
+BAD_CERT = 'bad_ca.crt'
```

Comment:
You don't need to define this as global constant, if you just use it inside one function.